src/lib/Repository/EventSubscriber/NameSchemaSubscriber.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Core\Repository\EventSubscriber;
+
+use Ibexa\Contracts\Core\Event\ResolveUrlAliasSchemaEvent;
+use Ibexa\Core\FieldType\FieldTypeRegistry;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+final class NameSchemaSubscriber implements EventSubscriberInterface
```

Comment:
Technically internal. Reasoning: only the event should be consumed by 3rd party in a similar manner, but the subscriber itself should not be used by 3rd party, they should create their own Subscribers if needed.
```suggestion
/**
 * @internal
 */
final class NameSchemaSubscriber implements EventSubscriberInterface
```